src/handlers/executors/contribution.ts

```diff
@@ -17,7 +17,7 @@ export const handleContributed = async ({
     data.toString()
   ) as [number, number[], string, string, string]
   const contributionRecord = Contribution.create({
-    id: idx.toString(),
+    id: `${extrinsic.extrinsic.hash.toString()}-${idx}`,
```

Comment:
should it use blockNumber as a prefix more make sense?

Reply:
I'm not very clear about the idx meaning. But it seems like the index of the events of the finalized extrinsic.